observability/monitoring/configuring-core-platform-monitoring/configuring-alerts-and-notifications.adoc

```diff
@@ -47,8 +47,8 @@ Alertmanager does not send notifications by default. It is strongly recommended
 .Additional resources
 
 * xref:../../../observability/monitoring/about-ocp-monitoring/key-concepts.adoc#sending-notifications-to-external-systems_key-concepts[Sending notifications to external systems]
-* link:https://www.pagerduty.com/[PagerDuty] (PagerDuty official site)
-* link:https://www.pagerduty.com/docs/guides/prometheus-integration-guide/[Prometheus Integration Guide] (PagerDuty official site)
+* link:https://www.pagerduty.com/[PagerDuty official site]
+* link:https://www.pagerduty.com/docs/guides/prometheus-integration-guide/[Prometheus Integration Guide for PagerDuty]
```

Comment:
```suggestion
* link:https://www.pagerduty.com/docs/guides/prometheus-integration-guide/[Prometheus Integration Guide] (PagerDuty documentation)
```

Reply:
My comments are based on the [Contributor Guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute) content.